scripts/Install

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+# Program information
+prog_name="BorgBackup"
+version="1.1.10"
+signature="BSD License (BSD)"
+git_url="https://github.com/borgbackup/borg"
+installation_path=/usr/local/bin
+
+# System information
+SYS=$(uname -s | awk '{print tolower($0)}')
+OS=$(awk '/DISTRIB_ID=/' /etc/*-release | sed 's/DISTRIB_ID=//' | tr '[:upper:]' '[:lower:]')
+ARCH=$(uname -m | sed 's/x86_//;s/i[3-6]86/32/')
+VERSION=$(awk '/DISTRIB_RELEASE=/' /etc/*-release | sed 's/DISTRIB_RELEASE=//' | sed 's/[.]0/./')
```

Comment:
I fear that this likely will break on some systems.

Did you try this on some linux systems, freebsd and mac osx?